module.txt

```diff
@@ -8,6 +8,10 @@
         {
             "id": "Health",
             "minVersion": "2.0.0-SNAPSHOT"
+        },
+        {
+            "id": "StructureTemplates",
+            "minVersion": "1.0.0"
         }
```

Comment:
I would have expected the dependency to be the other way around :thinking: ST depends on MB, because STs can provide a multi-block definition when spawning a building, but MB does not need to know whether the multi-block definition originates from an ST or something else...